iset.mm

```diff
@@ -142195,40 +142195,412 @@ Dedekind cut is that it is inhabited (bounded), rounded, disjoint, and
         DVMZXDWSHWIXEXAWHXCWRWDDWGUIUKXEXBVTWQWDDGVKVPVOVQSUAUPUCUPVRVS $.
     $}
 
-    $d A q r x y $.  $d B q r x y $.  $d L q r x y $.  $d U q r x y $.
-    $d ph q r x y $.
-    $( A Dedekind cut identifies a unique real number.  Similar to ~ df-inp
-       except that the the Dedekind cut is formed by sets of reals (rather than
-       positive rationals).  But in both cases the defining property of a
-       Dedekind cut is that it is inhabited (bounded), rounded, disjoint, and
-       located.  (Contributed by Jim Kingdon, 5-Jan-2024.) $)
-    dedekindicc $p |- ( ph -> E! x e. ( A [,] B ) (
-        A. q e. L q < x /\ A. r e. U x < r ) ) $=
-      ( wa vy cv clt wbr wral cicc co wrex wrmo wreu dedekindicclemlu wceq wcel
-      wi cap wn wfal cr ad4antr wss wb cin c0 simprll ad3antrrr simprlr simpllr
-      wo simprr simpr dedekindicclemeu iccssre syl2anc ad2antrr sseldd mpjaodan
-      reaplt mpbid inegd cc adantr recnd apti mpbird ancom anbi2i bitr4i anbi1i
-      anass bitri bicomi bitr3i imbi1i mpbi ex ralrimivva breq2 ralbidv anbi12d
-      breq1 rmo4 sylibr reu5 sylanbrc ) AHUBZBUBZUCUDZHFUEZXFGUBZUCUDZGEUEZTZBC
-      DUFUGZUHXLBXMUIZXLBXMUJABCDEFGHIJKLMNOPQRSUKAXLXEUAUBZUCUDZHFUEZXOXIUCUDZ
-      GEUEZTZTZXFXOULZUNZUAXMUEBXMUEXNAYCBUAXMXMAXFXMUMZXOXMUMZTZTZYAYBAYFXLTZT
-      ZXTTZYBUNYGYATZYBUNYJYBXFXOUOUDZUPZYJYLYJYLTZXFXOUCUDZUQXOXFUCUDZYNYOTCDX
-      FXOEFGHACURUMZYHXTYLYOIUSADURUMZYHXTYLYOJUSAFXMUTZYHXTYLYOKUSAEXMUTZYHXTY
-      LYOLUSAXEFUMZHXMUHZYHXTYLYOMUSAXIEUMZGXMUHZYHXTYLYONUSAUUAXEXIUCUDZGFUHVA
-      HXMUEZYHXTYLYOOUSAUUCUUEHEUHVAGXMUEZYHXTYLYOPUSAFEVBVCULZYHXTYLYOQUSAUUEU
-      UAUUCVHUNGXMUEHXMUEZYHXTYLYORUSACDUCUDZYHXTYLYOSUSYIYDXTYLYOAYDYEXLVDZVEY
-      IXLXTYLYOAYFXLVIZVEYIYEXTYLYOAYDYEXLVFZVEYIXTYLYOVGYNYOVJVKYNYPTCDXOXFEFG
-      HAYQYHXTYLYPIUSAYRYHXTYLYPJUSAYSYHXTYLYPKUSAYTYHXTYLYPLUSAUUBYHXTYLYPMUSA
-      UUDYHXTYLYPNUSAUUFYHXTYLYPOUSAUUGYHXTYLYPPUSAUUHYHXTYLYPQUSAUUIYHXTYLYPRU
-      SAUUJYHXTYLYPSUSYIYEXTYLYPUUMVEYIXTYLYPVGYIYDXTYLYPUUKVEYIXLXTYLYPUULVEYN
-      YPVJVKYNYLYOYPVHZYJYLVJYNXFURUMXOURUMYLUUNVAYNXMURXFAXMURUTZYHXTYLAYQYRUU
-      OIJCDVLVMZVEZYIYDXTYLUUKVNVOYNXMURXOUUQYIYEXTYLUUMVNVOXFXOVQVMVRVPVSYJXFV
-      TUMXOVTUMYBYMVAYJXFYJXMURXFAUUOYHXTUUPVNZYIYDXTUUKWAVOWBYJXOYJXMURXOUURYI
-      YEXTUUMWAVOWBXFXOWCVMWDYJYKYBYJYGXKXHXTTZTZTZYKYJAYFXKTZTZUUSTZUVAYJUVCXH
-      TZXTTUVDYIUVEXTYIAUVBXHTZTUVEYHUVFAYHYFXKXHTZTUVFXLUVGYFXHXKWEWFYFXKXHWIW
-      GWFAUVBXHWIWGWHUVCXHXTWIWJUVDYGXKTZUUSTUVAUVCUVHUUSUVHUVCAYFXKWIWKWHYGXKU
-      USWIWJWJUUTYAYGUUTUVGXTTYAXKXHXTWIUVGXLXTXKXHWEWHWLWFWJWMWNWOWPXLXTBUAXMY
-      BXHXQXKXSYBXGXPHFXFXOXEUCWQWRYBXJXRGEXFXOXIUCWTWRWSXAXBXLBXMXCXD $.
+    ${
+      $d A q r x y $.  $d B q r x y $.  $d L q r x y $.  $d U q r x y $.
+      $d ph q r x y $.
+      $( Lemma for ~ dedekindicc .  The result, except that we merely show it
```

Comment:
"The result" of what?

Reply:
Hmm, interesting, the overall context sorta answers that but when I look at the sentence more closely I realize that "the result" refers both to the theorem dedekindicc and to the value `x`, in different parts of it.

I guess we can go with "Same as ~ dedekindicc , except that we merely show ` x ` to be an element of ` ( A [,] B ) `"